src/chapters/aspdotnet-model-classes/data-layer.rst

```diff
@@ -0,0 +1,284 @@
+.. _data-layer:
+
+Models and Data 
+===============
+
+In order to work with data, we need to add another element to our MVC application. Say for example,
+we want to do things like remove an event from our list. Well, if two events both have the same name, 
+how might we identify which of those items to delete? We can't yet. So we need to tweak how we store
+event data. 
```

Comment:
This a confusingly worded paragraph. Maybe rephrase, "We need to add another element to our MVC application to effectively work with data. For example, if we want to remove an event from the list, but two events have the same name, we would not be able to identify which event we want to remove."